modules/efk-logging-about-crd.adoc

```diff
@@ -3,24 +3,24 @@
 // * logging/efk-logging.adoc
 
 [id="efk-logging-configuring-crd-{context}"]
-= About the Cluster logging CRD
+= About the Cluster Logging Custom Resource Definition
 
 The Cluster Logging Operator Custom Resource Definition (CRD) defines a complete cluster logging deployment 
 that includes all the components of the logging stack to collect, store and visualize logs. 
 
 You should never need to modify this CRD. To make changes to your deployment, create and modify a specific Custom Resource (CR).
 Instructions for creating or modifying a CR are provided in this documentation as appropriate.
 
-The following is an example of a typical Custom Resource Definition for cluster logging.
+The following is an example of a typical Custom Resource for cluster logging.
 
 [id="efk-logging-configuring-about-sample-{context}"]
-.Sample ClusterLogging CRD
+.Sample ClusterLogging CR
```

Comment:
@anpingli I changed this to Sample Cluster Logging CR. This is CR, not the CRD, I believe.